.github/workflows/build-fuzz-reusable.yml

```diff
@@ -33,6 +33,11 @@ jobs:
     - name: Install pip packages
       run: pip install -r third_party/requirements.txt
 
+    - name: Install apt packages
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y libhwloc-dev
```

Comment:
this job runs now on self-hosted runner with no sudo - libhwloc should be there already - you can remove this

Reply:
Done.